.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,6 +1,6 @@
 name: Bug Report
 description: File a bug report
-labels: ["t/bug", "s/needs-verification"]
```

Comment:
Since our only filter for triage is now basically going to be: is it on a milestone? We could opt for no initial label at all and add it during triage. That way it's also clear that we actually identified it as a bug. 

Thoughts?

Reply:
yea, this seems implicit. If it doesn't have "verified" than it needs verification

Reply:
But same for bug? Should we only label it as a bug if we actually find it to be a bug instead of applying that simply when the issue is opened?

Reply:
I think because it's the "bug" template then labeling as "bug" is reasonable. Of course sometimes it turns out to be wrong, but I'd consider it a strong indicator.